app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    separate = command.split()
+    old_file = separate[1]
+    new_file = separate[2]
+    if len(separate) == 3 and separate[0] == "cp" and old_file != new_file:
```

Comment:
The length check should be done before assigning old_file and new_file to avoid potential IndexErrors.